src/components/Pages/Search/index.js

```diff
@@ -1,46 +1,160 @@
 import React, { useState, useEffect  } from 'react'
 import { useRouteData, Head } from 'react-static';
 import { useIntl } from 'react-intl';
-import { events as i18n } from 'js/i18n/definitions';
+import { search as i18n } from 'js/i18n/definitions';
 
 import PageHeader from 'components/PageHeader';
 import SearchResult from 'components/Pages/Search/searchResult.js'
+import { searchWorker } from 'js/helpers/searchWorker'
+import { loader } from 'js/animations/loader';
 
-/* - - - TO DO: Finish Janis Search Page
-  - Issue: https://github.com/cityofaustin/techstack/issues/4285
-  // import { searchWorker } from 'js/helpers/searchWorker'
-  // import { loader } from 'js/animations/loader';
-*/
 
 const SearchPage = () => {
 
-  const { searchIndex } = useRouteData();
+  const intl = useIntl();
+  let { searchIndex } = useRouteData();
```

Comment:
is there a reason `searchIndex` needs to be `let` instead of `const` here? Is it being written to somewhere I'm just not seeing?

Reply:
Nope. Must have changed it at some point before cleaning up. But it's const now!🌟